rust-toolchain

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2021-09-01"
+channel = "nightly-2021-11-01"
```

Comment:
```suggestion
channel = "nightly-2021-09-10"
```

This way it matches https://github.com/paritytech/substrate/blob/0465b0bb407fe36ea03ae35d5138633e23119aac/shell.nix#L8